config.yaml

```diff
@@ -1,5 +1,19 @@
 - destination: &dest syn12177492
 - datasets:
+    - biodomains:
+          files:
+            - name: biodomains
+              id: syn44151254.1
+              format: csv
+          final_format: json
+          custom_transformations: 1
+          provenance:
+            - syn44151254.1
+          column_rename:
+            ensg: ensembl_gene_id
+            goterm_name: goterms
```

Comment:
nitpicking a little... could this maybe be `go_terms` instead of `goterms`?

Reply:
@jaclynbeck-sage . Something we do on our team is, maybe that'll help you.

Nit: some nitpicky comment

Reply:
done!